code/modules/library/skill_learning/generic_skillchips/misc.dm

```diff
@@ -0,0 +1,159 @@
+
+/obj/item/skillchip/brainwashing/on_activate(mob/living/carbon/user, silent = FALSE)
+	to_chat(user, span_danger("You get a pounding headache as the chip sends corrupt memories into your head!"))
+	user.adjustOrganLoss(ORGAN_SLOT_BRAIN, 20)
+	. = ..()
+
+/obj/item/skillchip/chefs_kiss
+	name = "K1SS skillchip"
+	auto_traits = list(TRAIT_CHEF_KISS)
+	skill_name = "Chef's Kiss"
+	skill_description = "Allows you to kiss food you've created to make them with love."
+	skill_icon = "cookie"
+	activate_message = span_notice("You recall learning from your grandmother how they baked their cookies with love.")
+	deactivate_message = span_notice("You forget all memories imparted upon you by your grandmother. Were they even your real grandma?")
+
+/obj/item/skillchip/intj
+	name = "Integrated Intuitive Thinking and Judging skillchip"
+	auto_traits = list(TRAIT_REMOTE_TASTING)
+	skill_name = "Mental Flavour Calculus"
+	skill_description = "When examining food, you can experience the flavours just as well as if you were eating it."
+	skill_icon = FA_ICON_DRUMSTICK_BITE
+	activate_message = span_notice("You think of your favourite food and realise that you can rotate its flavour in your mind.")
+	deactivate_message = span_notice("You feel your food-based mind palace crumbling...")
+
+/obj/item/skillchip/drunken_brawler
+	name = "F0RC3 4DD1CT10N skillchip"
+	auto_traits = list(TRAIT_DRUNKEN_BRAWLER)
+	skill_name = "Drunken Unarmed Proficiency"
+	skill_description = "When intoxicated, you gain increased unarmed effectiveness."
+	skill_icon = "wine-bottle"
+	activate_message = span_notice("You honestly could do with a drink. Never know when someone might try and jump you around here.")
+	deactivate_message = span_notice("You suddenly feel a lot safer going around the station sober... ")
+
+/obj/item/skillchip/master_angler
+	name = "Mast-Angl-Er skillchip"
+	auto_traits = list(TRAIT_REVEAL_FISH, TRAIT_EXAMINE_FISHING_SPOT, TRAIT_EXAMINE_FISH, TRAIT_EXAMINE_DEEPER_FISH)
+	skill_name = "Fisherman's Discernment"
+	skill_description = "Lists fishes when examining a fishing spot, gives a hint of whatever thing's biting the hook and more."
+	skill_icon = "fish"
+	activate_message = span_notice("You feel the knowledge and passion of several sunbaked, seasoned fishermen burn within you.")
+	deactivate_message = span_notice("You no longer feel like casting a fishing rod by the sunny riverside.")
+
+	actions_types = list(/datum/action/cooldown/fishing_tip)
+
+/datum/action/cooldown/fishing_tip
+	name = "Dispense Fishing Tip"
+	desc = "Recall a pearl of wisdom about fishing."
+	button_icon = 'icons/hud/radial_fishing.dmi'
+	button_icon_state = "river"
+	background_icon_state = "bg_default"
+	overlay_icon_state = "bg_default_border"
+	cooldown_time = 2.5 SECONDS //enough time to skim through tips.
+
+/datum/action/cooldown/fishing_tip/Activate(atom/target_atom)
+	. = ..()
+	if(!.)
+		return
```

Comment:
i dont think this would work. reason being that the parent doesnt actually return anything